kentikapi/client_integration_test.go

```diff
@@ -417,3 +425,110 @@ func timePtr(v time.Time) *time.Time {
 func durationPtr(v time.Duration) *time.Duration {
 	return &v
 }
+
+func TestGRPCClient_GetAgent(t *testing.T) {
+	tests := []struct {
+		name    string
+		id      string
+		res     *pb.GetAgentResponse
+		errCode codes.Code
+		errMsg  string
+	}{
+		{
+			"invalid request with wrong id",
+			"1000",
+			nil,
+			codes.InvalidArgument,
+			"cannot find ID",
+		},
+		{
+			"valid request with valid id",
```

Comment:
```suggestion
			"valid request, agent returned",
```
Why? To indicate the behaviour of test server to better describe the test.

Reply:
Done